README.md

```diff
@@ -1,49 +1,207 @@
 # Ballerina Redis Connector
 
-[![Build](https://github.com/ballerina-platform/module-ballerinax-redis/workflows/CI/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/actions?query=workflow%3ACI)
-[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-redis/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-redis)
+[![Build](https://github.com/ballerina-platform/module-ballerinax-redis/actions/workflows/ci.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/actions/workflows/ci.yml)
 [![Trivy](https://github.com/ballerina-platform/module-ballerinax-redis/actions/workflows/trivy-scan.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/actions/workflows/trivy-scan.yml)
+[![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinax-redis/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinax-redis)
+[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-redis/actions/workflows/build-with-bal-test-graalvm.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/actions/workflows/build-with-bal-test-graalvm.yml)
 [![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerinax-redis.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/commits/master)
-[![GraalVM Check](https://github.com/ballerina-platform/module-ballerinax-redis/actions/workflows/build-with-bal-test-native.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinax-redis/actions/workflows/build-with-bal-test-native.yml)
-[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+[![GitHub Issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-library/module/redis.svg?label=Open%20Issues)](https://github.com/ballerina-platform/ballerina-library/labels/module%2Fredis)
+
+[Redis](https://redis.io/) is an open-source, in-memory data structure store that can be used as a database,
+cache, and message broker. It supports various data structures such as strings, hashes, lists, sets, and more.
+
+The `ballerinax/redis` package offers APIs to connect to Redis servers and to manipulate key-value data, providing
+functionality to utilize Redis both as a database and cache. The current connector is compatible Redis server
+versions up to 7.2.x.
+
+## Setup guide
+
+To use the Redis connector, you need to have a Redis server running and accessible. For that, you can either install
+Redis locally or use a cloud-based Redis service.
+
+### Setting up a Redis server locally
+
+#### Step 1: Install Redis Server
```

Comment:
We can use sentence case for these headings as well

Reply:
I think since `Redis` is kind of a technology name, we should make it capitalize.

Reply:
@ayeshLK correct, but I think he is focused on the word `Server` in here. Updated the similar places with  https://github.com/ballerina-platform/module-ballerinax-redis/pull/220/commits/75e3931ebc1902d7873c7d3281b80d4bb93219d1